addons/delivery/static/src/js/location_selector/location_selector_dialog/location_selector_dialog.js

```diff
@@ -11,7 +11,9 @@ export class LocationSelectorDialog extends Component {
     static components = { Dialog, LocationList, MapContainer };
     static template = 'delivery.locationSelector.dialog';
     static props = {
-        orderId: Number,
+        orderId: {type: Number, optional: true},
+        parentModel: {type: String, optional: true},
+        parentId: {type: Number, optional: true},
```

Comment:
Since we are in the backend, why can't we directly assign the partner to the right field from JS? We can adapt the eCommerce component to handle the location if that's the only issue.

I think it would be simpler if the location selector would only return the chosen location and then let the components that opens it deal with it.

Reply:
These props are not used to set the partner. However, they're passed to the `_get_locations` request to be used in the cases mentioned in the other comment.